<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorseDecoderSettings</class>
 <widget class="QDialog" name="MorseDecoderSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morse code settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLabel" name="BufferLastL">
     <property name="text">
      <string>Last signals from buffer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="MorseLengthFactorV">
     <property name="text">
      <string>50</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="AutoCalcT">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="MorseLastSignalsL">
     <property name="text">
      <string>Last signals to calc</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="PauseShortFactorT">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="PauseLongFactorT">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="MorseCalcB">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="PauseLongFactorL">
     <property name="text">
      <string>Long pause threshold</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="MorseLengthFactorL">
     <property name="text">
      <string>Short/long threshold</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="MorseLastSignalsT">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="GenVolumeL">
     <property name="text">
      <string>Generator volume</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="WPMStandardL">
     <property name="text">
      <string>Speed standard</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QLineEdit" name="TransmitBufLatencyT"/>
   </item>
   <item row="10" column="1">
    <widget class="QSlider" name="GenVolumeS">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QComboBox" name="WPMStandardT">
     <item>
      <property name="text">
       <string>PARIS (50)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CODEX (60)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="GenVolumeV">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="BufferLastT">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="PauseShortFactorL">
     <property name="text">
      <string>Short pause threshold</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="AutoCalcC">
     <property name="text">
      <string>Auto calculate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="TestModeC">
     <property name="text">
      <string>Test mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="MorseLengthFactorS">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QLineEdit" name="MuteTransmitT"/>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="MuteTransmitC">
     <property name="text">
      <string>Mute transmit delay (ms)</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="TransmitBufLatencyL">
     <property name="text">
      <string>Transmit display latency (ms)</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="TransmitFadeL">
     <property name="text">
      <string>Signal fade value (ms)</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="QLineEdit" name="TransmitFadeT"/>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="SoundFreqC">
     <property name="text">
      <string>Sound frequency</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QLineEdit" name="SoundFreqT"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>MorseLastSignalsT</tabstop>
  <tabstop>MorseLengthFactorS</tabstop>
  <tabstop>PauseShortFactorT</tabstop>
  <tabstop>PauseLongFactorT</tabstop>
  <tabstop>MorseCalcB</tabstop>
  <tabstop>AutoCalcC</tabstop>
  <tabstop>AutoCalcT</tabstop>
  <tabstop>BufferLastT</tabstop>
  <tabstop>TestModeC</tabstop>
  <tabstop>WPMStandardT</tabstop>
  <tabstop>GenVolumeS</tabstop>
  <tabstop>SoundFreqC</tabstop>
  <tabstop>SoundFreqT</tabstop>
  <tabstop>MuteTransmitC</tabstop>
  <tabstop>MuteTransmitT</tabstop>
  <tabstop>TransmitBufLatencyT</tabstop>
  <tabstop>TransmitFadeT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
